interlok-core/src/test/java/com/adaptris/core/ftp/RelaxedFtpConsumerTest.java

```diff
@@ -75,7 +75,7 @@ public class RelaxedFtpConsumerTest extends RelaxedFtpConsumerCase {
   public void setUp() throws Exception {
     consumer = new RelaxedFtpConsumer();
 
-    MockitoAnnotations.initMocks(this);
+    MockitoAnnotations.openMocks(this);
```

Comment:
You're supposed to close the mock as well normally.
Something like:

```
  private AutoCloseable closeable;
  @BeforeEach
  void initService() {
    ...
    closeable = MockitoAnnotations.openMocks(this);
    ...
  }

  @AfterEach
  void closeService() throws Exception {
    closeable.close();
  }
```

But maybe it's not a bid deal.
The alternative could be to use `testImplementation("org.mockito:mockito-junit-jupiter:$mockitoVersion")` to auto open and close mocks

Reply:
okay; added where necessary